### ISSUE TITLE
Bank OCBC Indonesia

### ISSUE THREAD

**Issue by koenvanhollebeke** [osmlab/name-suggestion-index#9994]
Please add the following bank and ATM brand in Indonesia:

amenity=bank
brand=OCBC
brand:wikidata=Q19724214
name=OCBC
official_name=Bank OCBC NISP

amenity=atm
brand=OCBC
brand:wikidata=Q19724214
official_name=Bank OCBC NISP
operator=OCBC
operator:wikidata=Q2882611

Thanks.

**Comment by Snowysauce** [comment]
Added OCBC to `brands/amenity/bank`. The ATM entry/tags for OCBC will be automatically generated by a template based on the entry in `brands/amenity/bank`.

**Comment by koenvanhollebeke** [comment]
Hi,
Thanks for adding the bank. 

Just a question: Some time ago you added some ferry companies for me - I can see them now on the NSI webpage. I was expecting that this would give me in OSM the option of attributing a ferry route to for example Pelni (for example: https://www.openstreetmap.org/edit?editor=id&relation=18055577) I tried to add the network=Pelni but nothing happens - no Icon, no automatical update of network:wikidata tag, ... Am I too quick, ie OSM hasn't updated the NSI info yet?
Kind regards,
Koen

**Comment by Snowysauce** [comment]
> Am I too quick, ie OSM hasn't updated the NSI info yet?

There was a bit of a delay between the update to NSI.guide and when the update became available to iD/other editors as we worked through some logistics. It then took a few more hours before iD picked up the new release, but the update went live on iD (at least for me) beginning sometime late yesterday (US Eastern time).

When I clicked on the link you provided just now, iD instantly auto-suggested the Pelni network tag changes from NSI. I'm not really familiar with OSM's server setup, but perhaps there are some servers outside the US that haven't yet loaded in the new release data. If that's the case, I'd expect them to grab the update very soon.
